app/backend/src/couchers/migrations/versions/8c878b177151_add_hosting_location_to_host_request.py

```diff
@@ -0,0 +1,43 @@
+"""add hosting location to host request
+
+Revision ID: 8c878b177151
+Revises: 14585a4e1868
+Create Date: 2025-07-07 21:01:50.381450
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from geoalchemy2 import Geometry
+
+# revision identifiers, used by Alembic.
+revision = "8c878b177151"
+down_revision = "14585a4e1868"
```

Comment:
Please update to `down_revision = "d345174cb002"`

Reply:
Actually don't worry about these, I can update it when it's ready to go (given `develop` is getting lots of migrations right now)